### PR TITLE
Add reverse proxy with CORS support for riesgos-wps and geoserver

### PR DESCRIPTION
**Pull request by nbrinckm**
This pull request here add a first set of proxy rules for the riesgos-wps & the according geoserver.

### CODE REVIEW

**Comment by nbrinckm** [comment]
Honestly I didn't got the proxy location work, so I skip it for now.

File: .env [hunk 1]
```diff
@@ -48,7 +48,7 @@ riesgosWpsGeoserverPassword=geoserver
 # of the docker compose.
 # We also have the same for port & protocol of the resulting url.
 riesgosWpsAccessServerHost=localhost
-riesgosWpsAccessServerPort=8082
+riesgosWpsAccessServerPort=80
```
Comment (MichaelLangbein) [inline .env:51]: Fine by me - that port's still free - also with DLR's docker-compose taken into consideration.

**Comment by MichaelLangbein** [comment]
> Honestly I didn't got the proxy location work, so I skip it for now.

What location would that be? Something important?

**Review by MichaelLangbein — APPROVED** [review]
Looks alright to me.
If further discussion is required, feel free to re-open!

**Comment by nbrinckm** [comment]
> > Honestly I didn't got the proxy location work, so I skip it for now.
> 
> What location would that be? Something important?

It was what you mentioned here https://github.com/riesgos/buildall/issues/8#issuecomment-1560623453 
I'm not sure if I need some more settings, or I'm just stupid, but I didn't get that work.

**Pull request merged**